core/src/view/flyTo.cpp

```diff
@@ -0,0 +1,85 @@
+#include "view/flyTo.h"
+
+#include "util/mapProjection.h"
+#include "view/view.h"
+#include <cmath>
+#include "log.h"
+
+namespace Tangram {
+
+float getMinimumEnclosingZoom(double aLng, double aLat, double bLng, double bLat, const View& view, float buffer) {
```

Comment:
Is `getMinimumEnclosingZoom` being used? Is this required?

Reply:
I wrote this function as part of my first implementation of `flyTo`. I don't think it's used now, but we'll need it when we enable moving the camera to a target area.